src/measure_theory/integral/limit_comparison.lean

```diff
@@ -0,0 +1,241 @@
+/-
+Copyright (c) 2022 David Loeffler. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: David Loeffler
+-/
+
+import measure_theory.integral.interval_integral
+
+import analysis.special_functions.exponential
+import analysis.special_functions.integrals
+import measure_theory.integral.integral_eq_improper
+
+
+/-!
+# Limit comparison test for integrals
+
+As easy special cases of general theorems in the library, we prove the following tests
+for integrability:
+
+* If f is continuous on [a,∞), for some a ∈ ℝ, and there exists b > 0 such that
+  f(x) = O(exp(-b x)) as x → ∞, then f is integrable on (a, ∞).
+
+-/
+
+noncomputable theory
+open real interval_integral measure_theory set filter
+open_locale classical topological_space ennreal
+
+namespace limit_comparison
+
+/-- exp(-b x) is integrable on (a, X] for any finite a, X. -/
+lemma exp_neg_finite_integrable (a : ℝ) (b : ℝ) (X : ℝ):
+  integrable_on (λ x : ℝ, exp(-b * x) ) (set.Ioc a X) :=
+begin
+  apply continuous.integrable_on_Ioc,
+  apply continuous.exp,
+  apply continuous.mul,
+  apply continuous_const,
+  apply continuous_id
+end
```

Comment:
This looks like you can also write
```suggestion
(continuous_const.mul continuous_id).exp.integrable_on_Ioc
```
Maybe you need to supply some arguments to `exp` or `integrable_on_Ioc`. I didn't check.